### PR TITLE
Add Sam in the core team

### PR DESCRIPTION
**Pull request by fabpot**

### CODE REVIEW

File: contributing/code/core_team.rst [hunk 1]
```diff
@@ -84,7 +84,9 @@ Active Core Members
     Form_, Serializer_, DependencyInjection_, and HttpKernel_ components;
 
   * **Tobias Nyholm** (`Nyholm`) manages the official and contrib recipes
-    repositories.
+    repositories;
+
+  * **Samuel Roze** (`sroze`_) can merge into Messenger_ component.
```
Comment (javiereguiluz) [inline contributing/code/core_team.rst]: Minor typo: `Roze` -> `Rozé`

**Review by javiereguiluz — APPROVED** [review]

File: contributing/code/core_team.rst (hunk 1 above)
Comment (fabpot) [reply contributing/code/core_team.rst]: fixed

**Comment by javiereguiluz** [comment]
Thanks Fabien.

**Comment by sroze** [comment]
Travis complained about the missing target though. I guess you need to add that at the end of the file:
```
 .. _`sroze`: https://github.com/sroze
```

:)

**Comment by javiereguiluz** [comment]
Thanks! Fixed in 28377f7b2ffa750e46318f1c0d8c917f591e66e4

**Pull request merged**